paimon-core/src/main/java/org/apache/paimon/Snapshot.java

```diff
@@ -184,7 +192,8 @@ public Snapshot(
             @Nullable Long totalRecordCount,
             @Nullable Long deltaRecordCount,
             @Nullable Long changelogRecordCount,
-            @Nullable Long watermark) {
+            @Nullable Long watermark,
+            @Nullable String statistics) {
```

Comment:
If the partition-level statistics are supported, how to retrieve those?

Reply:
Plan to add a new field named `partitionStatistics` (a manifestList link) when going to support partition-level statistics

Reply:
both `statistics` and `partitionStatistics` are related to statistics, why we split into two?